pages/docs/engine/ra-optimisation/branch-and-bound-optimisation/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Branch and bound optimisation
+permalink: /docs/engine/ra-optimisation/branch-and-bound/
+hide: true
+feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
+tags: [Docs, Branch & Bound RAO]
+---
+
+### Overview
+
+The RAO optimisation problem is of such complexity that its overall optimum cannot be found, on real size data sets, in a reasonnable computation time. Therefore, FARAO does not offer only one RAO algorithm, but several, each one having its pros and cons in terms of:
+
+- handling of remedial actions (Are topological actions optimized? Are costly remedial actions such as redispatching and counter-trading taken into account?),
+
+- optimality measurement and optimality criterion (In the scope of its problem, can the quality (i.e. proximity to the optimum) of the solution computed by the RAO be measured? If yes, is it the optimal solution or a close, « good enough » solution?),
+
+- computation time.
```

Comment:
I would have put that in the computation engine page (docs/engine/), to introduce the `Remedial action optimiser`. And just delete this page which seems redundant with the computatin engine page.